app/models/miq_expression.rb

```diff
@@ -1557,6 +1557,30 @@ def self._quick_search?(e)
     end
   end
 
+  def custom_attribute_columns
```

Comment:
WDYT about squashing these two methods into one? I"m not sure the `self._somewhat_private_class_method` style is one we want to continue to propagate, though it is used extensively in this class